ddtrace/llmobs/_llmobs.py

```diff
@@ -43,6 +47,13 @@
 log = get_logger(__name__)
 
 
+SUPPORTED_INTEGRATIONS = {
+    "bedrock": lambda: patch(botocore=True),
+    "langchain": lambda: patch(langchain=True),
+    "openai": lambda: patch(openai=True),
```

Comment:
I replaced `LLMObs.openai, LLMObs.bedrock, LLMObs.langchain` with this to allow users to specify string arguments (would be simpler than to make them have to use yet another `LLMObs` SDK object). Will need to change documentation accordingly